src/nuklear_font.c

```diff
@@ -1179,6 +1179,7 @@ nk_font_atlas_bake(struct nk_font_atlas *atlas, int *width, int *height,
     tmp = atlas->temporary.alloc(atlas->temporary.userdata,0, tmp_size);
     NK_ASSERT(tmp);
     if (!tmp) goto failed;
+	memset(tmp,0,tmp_size);
```

Comment:
Please tabs to spaces and then I'm merging. This is a leap forward for Nuklear. Thanks!